fe/fe-core/src/main/java/org/apache/doris/statistics/StatisticsAutoCollector.java

```diff
@@ -50,7 +50,7 @@ public class StatisticsAutoCollector extends StatisticsCollector {
 
     public StatisticsAutoCollector() {
         super("Automatic Analyzer",
-                TimeUnit.MINUTES.toMillis(Config.auto_check_statistics_in_minutes),
+                TimeUnit.SECONDS.toMillis(Config.auto_check_statistics_in_minutes),
```

Comment:
Should this config item auto_check_statistics_in_minutes changed to auto_check_statistics_in_seconds?